libraries/GCS_MAVLink/GCS.cpp

```diff
@@ -425,6 +425,20 @@ MAV_RESULT GCS::lua_command_int_packet(const mavlink_command_int_t &packet)
 
     return ch->handle_command_int_packet(packet, msg);
 }
+
+MAV_RESULT GCS::lua_command_long_packet(const mavlink_command_long_t &packet)
+{
+    // for now we assume channel 0. In the future we may create a dedicated channel
+    auto *ch = chan(0);
+    if (ch == nullptr) {
+        return MAV_RESULT_UNSUPPORTED;
+    }
+    // we need a dummy message for some calls
+    mavlink_message_t msg {};
+
+    return ch->try_command_long_as_command_int(packet, msg);
```

Comment:
Since this just calls into the command int packet what is the benefit of adding the new method vs doing the conversion in lua?

Reply:
Yep, that works well. I'll modify to do the ``command_long`` to ``command_int`` conversion in the lua script.

Reply:
> Since this just calls into the command int packet what is the benefit of adding the new method vs doing the conversion in lua?

Doing the conversion in LUA would need to do what we do in the C code, and that's not trivial; we have to come up with a best-guess frame for several incoming location-bearing commands, for example.